index.html

```diff
@@ -292,6 +292,11 @@ <h3>API</h3>
       to choose certificate for right domain. Also the <tt>hostname></tt> is used for
       hostname verification if verification is enabled by <tt>verify</tt>.
     </p>
+    <p>
+      <tt>alpn-protocols</tt> if specified, should be a list of alpn protocol
+      names that would be offered to the server. Protocol selected by the server
+      can be retrieved by <tt>get-selected-alpn-protocol</tt>.
+    </p>
```

Comment:
It would be good also to add description of the `cl+ssl:get-selected-alpn-protocol`

Reply:
Ok.